packages/web/src/components/drawer/Drawer.tsx

```diff
@@ -207,7 +194,7 @@ const DraggableDrawer = ({
       movement: [, my],
       memo = currentTranslation()
     }) => {
-      const height = getHeight()
+      const height = getHeight(contentRef)
```

Comment:
Okay this one I'm almost certain is going to fire more frequently than frame renders and might cause a perf issue.
And if that's the case, this might be a little complicated to do correctly with rAF. Open to discussing some options.

Reply:
so this isn't actually any new behavior - it was just a refactor that part of from a previous PR came in. i'm down to discuss this one more though - can we do that after this change goes in (which is pretty minimal now)?

Reply:
Oh I completely glossed over the code removal and thought you had added the call to `getBoundingClientRect`. Probably fine to leave alone for now.